packages/okta-react-native/README.md

```diff
@@ -50,145 +40,219 @@ After you have created the application there are two more values you will need t
+## Setup
 
-1. Open the iOS folder in Xcode
-2. Open `info.plist`, and create a new row called `URL types`
-3. Expand `URL types`, and expand `Item 0`, then choose `URL Schemes`
-4. Under `URL Schemes`, add your scheme to `Item 0`
+### iOS Setup
 
-#### Configuring Android
+#### Install Okta Open ID Connect iOS
+This library depends on the native [Okta OIDC iOS](https://github.com/okta/okta-oidc-ios) library. This library is not distributed as part of the React Native library to keep your dependency management consistent. 
 
-1. In `android/app/src/main`, open `AndroidManifest.xml`
-2. Add the following intent filter below the last intent filter that is already listed:
-```
-<intent-filter android:label="filter_react_native">
-  <action android:name="android.intent.action.VIEW" />
-  <category android:name="android.intent.category.DEFAULT" />
-  <category android:name="android.intent.category.BROWSABLE" />
-  <data android:scheme="yourScheme"/>
-</intent-filter>
-```
+There are three ways to add Okta OIDC iOS to your dependencies:
 
-#### [Optional] Routing
+1. **CocoaPods**
+   With [CocoaPods](https://guides.cocoapods.org/using/getting-started.html), add the following line to
+   your `Podfile`:
+       
+		pod 'OktaOidc'
+			 
+   Then run `pod install`.
 
-If you would like to configure routing and navigation, you will need to install and configure [React Navigation](https://reactnavigation.org/en/)
+2. **Carthage**
+	 To integrate this SDK into your Xcode project using [Carthage](https://github.com/Carthage/Carthage), specify it in your `Cartfile`:
+ 		 
+		github "okta/okta-oidc-ios"		 
 
-### Expo Users
+	 Then run `carthage update --platform iOS`.
 
-Assuming you're using an app created with `create-react-native-app`, modify your `app.json` to add a `scheme`:
+   Drag and drop `OktaOidc.framework` from `ios/Carthage/Build/iOS` to `Frameworks` in Xcode.
 
-```javascript
-{
-  "expo": {
-    "sdkVersion": "25.0.0",
-    "scheme": "com.oktapreview.{yourOrg}"
-  }
-}
-```
+   Add a copy files build step for `OktaOidc.framework`: open Build Phases on Xcode, add a new "Cope Files" phase, choose "Frameworks" as destination, add `OktaOidc.framework` and ensure "Code Sign on Copy" is checked.			
+
+3. **Static Library**
+ 	 This requires linking the Okta OIDC iOS and your project, and including the headers. 
+	 Suggested configuration:
+
+   1. Create an XCode Workspace.
+   2. Add `okta-oidc.xcodeproj` to your Workspace.
+   3. Include `OktaOidc.framework` as a linked framework for your target (in the "General -> Linked Framework and
+      Libraries" section of your target).
+   4. Add `okta-oidc/okta-oidc/Okta` to your search paths of your target ("Build Settings -> "Header Search
+      Paths").
+
+#### Swift Configuration
+Since React Native uses Objective-C, and Okta React Native library is a swift wrapper, you will need to have at least one Swift file in your iOS project for the project to compile. To do this, follow the following steps:
```

Comment:
Fairly speaking you don't need to add dummy swift file. I would write something like:
`Add project setting SWIFT_VERSION = 4.2. To do this open Build Settings in Xcode, select Edit -> Add Build setting -> Add User-Defined Setting. Specify SWIFT_VERSION and 4.2 as setting name and value correspondently.`

Reply:
I have to test this to verify, will update the doc once it's being tested!